entity-designer/entity-designer.md

```diff
@@ -152,10 +152,51 @@ private Long id;
 
 ## Extract to MappedSuperclass
 
-As your application grows and the JPA model evolves, you may realize that certain attributes are common among multiple entities and should be extracted to a `@MappedSuperclass` for better code organization. JPA Buddy can easily extract attributes along with JPA annotations to MappedSuperclass and build a well-designed entities' hierarchy.
+As your application grows and the JPA model evolves, you may realize that certain attributes are common among multiple entities and should be extracted to a `@MappedSuperclass` for better code organization. JPA Buddy can easily extract methods and attributes along with their JPA annotations to the MappedSuperclass and build a well-designed entity hierarchy.
+
+To achieve this, navigate to the Entity Designer tab and access the Actions menu. From there, select "Extract to MappedSuperclass" as shown below:
+
+![ed-extract-to-mappedsuperclass.png](img/ed-extract-to-mappedsuperclass.png)
+
+Next choose the desired attributes and methods to include in the MappedSuperclass. JPA Buddy will automatically apply the existing annotations to the extracted attributes.
```

Comment:
When methods are selected, and some attributes are used in them JPA Buddy will highlight them. Maybe we need to mention it as well.